snooty/postprocess.py

```diff
@@ -365,6 +366,13 @@ def enter_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
                     )
                 return
 
+        # Check for duplicate inclusions in this root page context.
```

Comment:
[minor nitpick] can we add this to the description for `bound_included_AST` as well?